Document/0x05b-Basic-Security_Testing.md

```diff
@@ -105,11 +105,59 @@ An extensive list of root detection methods is presented in the "Testing Anti-Re
 
 For a typical mobile app security build, you'll usually want to test a debug build with root detection disabled. If such a build is not available for testing, you can disable root detection in a variety of ways that will be introduced later in this book.
 
-#### Recommended Tools
+#### Recommended Tools - Android device
 
--- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1227>
+There are many tools and frameworks used throughout this guide to assess the security of Android applications. In the next sections, you will learn more about some of the commands and interesting use cases. Please check the official documentation for installation instructions of the following tools/APKs:
 
-##### adb
+- APK Extractor: App to extract APKs without root.
+- Frida server: Server for Frida, the dynamic instrumentation toolkit for developers, reverse-engineers, and security researchers. See [Frida](#Frida "Frida section") section below for more information.
+- Drozer agent: Agent for drozer, the framework that allows you to search for security vulnerabilities in apps and devices. See [Drozer](#Drozer "Drozer section") section below for more information.
+
```

Comment:
in iOS, 0x06b: we see that all tools are mentioned beforehand. Can ew do the same here so that it is in sync.

Reply:
In "Recommended Tips - iOS device" you can find the list of tools you can mainly install via Cydia. The same we have here, also a list of tools to install on your Android device. Or do you mean another thing or another section?

Reply:
nope, exactly that. in that cse: let's resolve, approve and merge